DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -292,14 +416,38 @@ function SetWatchFlags() {
         WatchFlag('MostWarehouseTroopsDead');
         break;
     case "04_NYC_UNATCOHQ":
+        WatchFlag('Shannon_Dead');
+        WatchFlag('M04MeetWalton_Played');
         if(RevisionMaps){
             bt = class'BingoTrigger'.static.Create(self,'AlexCloset',vectm(1725,-1062,-40),95,40);
             bt = class'BingoTrigger'.static.Create(self,'BathroomFlags',vectm(1130,-150,310),80,40);
+            bt.MakeClassProximityTrigger(class'#var(prefix)FlagPole');
+            bt = class'BingoTrigger'.static.Create(self,'PresentForManderley',vectm(960,234,297),350,60);
+            bt.MakeClassProximityTrigger(class'#var(prefix)JuanLebedevCarcass');
         } else {
             bt = class'BingoTrigger'.static.Create(self,'AlexCloset',vectm(1551.508301,-820.408875,-39.901726),95,40);
             bt = class'BingoTrigger'.static.Create(self,'BathroomFlags',vectm(240.180969,-385.104431,280.098511),80,40);
+            bt.MakeClassProximityTrigger(class'#var(prefix)FlagPole');
+            bt = class'BingoTrigger'.static.Create(self,'PresentForManderley',vectm(220,4,280),300,40);
+            bt.MakeClassProximityTrigger(class'#var(prefix)JuanLebedevCarcass');
         }
-        bt.MakeClassProximityTrigger(class'#var(prefix)FlagPole');
+
+
+        if(RevisionMaps){
```

Comment:
TODO: ChefCarcass Cremation for Revision maps

Reply:
Hmmm, no, this is some weird remnant that I guess I copied and pasted while working on PresentForManderley, since this is in the 04_NYC_UNATCOHQ map